apps/dapp/src/components/Pages/Core/DappPages/Dashboard/hooks/use-dashboardv2-txHistory.ts

```diff
@@ -78,58 +86,68 @@ const dashboardTypeToStrategyKey = (dType: DashboardType): StrategyKey => {
   }
 };
 
+const getTableHeaderOrderByType = (tableHeader?: TxHistoryTableHeader) => {
+  if (!tableHeader) return 'timestamp';
+  switch(tableHeader.name){
+    case 'Type':
+      return 'strategy__name';
+    case 'Date':
+      return 'timestamp';
+    case 'Debt Token':
+      return 'token__name';
+    case 'Borrow':
```

Comment:
The Borrow/Repay thing is confusing from a UX POV, clicking one column and seeing two sort indicators at the top. What if we instead had a single column "Amount" and in the amount, you concatenated the `amount` from the subgraph, with an indication of if it's borrowed or repayed, e.g. `90,000 borrowed` or `19,037 repaid`? So we sort by the amount in or out and append the activity with it.

I guess we need the design sign off on that. Wdyt?

Reply:
thx for pointing this out, went for [Lux option](https://discord.com/channels/847178511604252741/1131806977488588903/1169455310650486825)